docker/bench/bench_init.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+echo "$MINT_KEY" | base64 -d > /opt/libra/etc/mint.key
+/opt/libra/bin/ruben -a $AC_HOST -f /opt/libra/etc/mint.key -d $AC_DEBUG --metrics_server_address "0.0.0.0:14297" $@
```

Comment:
Just a note: after this PR merged, is it safe to delete the `-d` option from RuBen and also the `-d $AC_DEBUG` here?

Reply:
good point, removed `-d`